terratest/test/helper.go

```diff
@@ -0,0 +1,26 @@
+package test
+
+import (
+	"fmt"
+	"strings"
+	"sort"
+	"testing"
+	"github.com/gruntwork-io/terratest/modules/shell"
+)
+
+func Dig(t *testing.T, dnsServer string, dnsPort int, dnsName string) ([]string, error) {
```

Comment:
just comment: We have very similar function in k8gb. Soon, we should move into AbsaOSS/gopg and reuse